src/Domain.LinnApps/Requisitions/StoresFunction.cs

```diff
@@ -75,5 +75,15 @@ public Nominal GetNominal()
             }
             return null;
         }
+
+        public IList<string> GetTransactionStates(string fromOrOnto)
+        {
+            if (this.TransactionsTypes != null)
+            {
+                return this.TransactionsTypes.Select(t => t.TransactionDefinition?.GetTransactionStates(fromOrOnto)).SelectMany( s => s ).Distinct().OrderBy(s => s).ToList();
```

Comment:
can just do the selectmany on transactionTypes ?
probably need to wrap long line too